tests/update_state_test.py

```diff
@@ -0,0 +1,202 @@
+        # make all blue cards guessed
+        for card in game.board.cards:
+            if card.color == 'blue':
+                card.flipped = True
+
+        game.check_win()
+
+        assert game.winner_color == 'blue'
+        assert game.game_over is True
+
+class TestUpdateState:
+    '''Update_state sets guessing to True or False, then sets skip True first then False,
+    and finally, switches the team color and resets guess_value to zero'''
+    def test_guessing_is_true_under_right_conditions(self):
+        'If these three variables are as they are set here, guessing should be True'
+        game = Game()
+        game.guess_value = 1
+        game.code_number = 1
+        game.game_over = False
+
+        game.update_state()
+
+        assert game.guessing is True
+
+    def test_game_over_and_skip_are_true_makes_guessing_false(self):
+        'When game_over is True guessing is False. When skip is True, it is changed to False'
+        'Team color is changed from blue to red. Guess_value is reset'
+        game = Game()
+        game.guess_value = 1
+        game.code_number = 1
+        game.game_over = True
+        # need skip to be True so that self.guessing is not reset to True before test
+        game.skip = True
+        game.team = "blue"
+
+        game.update_state()
+
+        assert game.skip is False
+        assert game.guessing is False
+        assert game.team == "red" # team color should be switched if game.skip is False
+        assert game.guess_value == 0 # guess_value should reset if game.skip is False
+
+    def test_guessing_is_false_when_code_number_is_zero(self):
+        'If code_number is 0, guessing should be False'
+        'Team color is changed from red to blue. Guess_value is reset'
+        game = Game()
+        game.guess_value = 1
+        game.code_number = 0
+        game.game_over = False
+        # need skip to be True so that self.guessing is not reset to True before test
+        game.skip = True
+        game.team = "red"
+
+        game.update_state()
+
+        assert game.skip is False
+        assert game.guessing is False
+        assert game.team == "blue" # team color should be switched if game.skip is False
+        assert game.guess_value == 0 # guess_value should reset if game.skip is False
+
+    def test_guessing_is_false_when_guess_value_is_greater_than_one(self):
+        'If guess_value is more than one, guessing should be False'
+        game = Game()
+        game.guess_value = 2
+        game.code_number = 1
+        game.game_over = False
+        # need skip to be True so that self.guessing is not reset to True before test
+        game.skip = True
+
+        game.update_state()
+
+        assert game.skip is False
+        assert game.guessing is False
+
+    def test_skip_is_true_when_first_guessing_is_false(self):
+        'When guessing is false and skip is false, skip and guessing should be changed to True'
+        game = Game()
+        game.skip = False
+
```

Comment:
Add `assert game.guessing is False` before `game.update_state()` to document your assumption.